src/hmem_rocr.c

```diff
@@ -1054,6 +1063,100 @@ int rocr_dev_reg_copy_from_hmem(uint64_t handle, void *dest, const void *src,
 	return FI_SUCCESS;
 }
 
+bool rocr_is_dmabuf_supported(void)
+{
+	hsa_status_t hsa_ret;
+	bool dmabuf_support = false, dmabuf_kernel = false;
+
+	char *s;
+	if (!(s = getenv("FI_HMEM_ROCR_USE_DMABUF")) || !atoi(s)) goto out;
+
+	// HSA_AMD_SYSTEM_INFO_DMABUF_SUPPORTED = 0x204, we use the number instead of var name
+	// for backward compatibility reasons.
+	hsa_ret = hsa_ops.hsa_system_get_info((hsa_system_info_t) 0x204, &dmabuf_support);
+
```

Comment:
nitpick: here we usually don't add the empty line between the call and the return value checking.

Reply:
I'm assuming the return value checking clause after this should also have { at same line?